CRM/Extension/Info.php

```diff
@@ -141,6 +168,12 @@ public function parse($info) {
           );
         }
       }
+      elseif ($attr === 'requires') {
```

Comment:
i think it would more neat if parse() method organized in such way instead of this list of if/else :

```php
    foreach ($info as $attr => $val) {
      if (count($val->children()) == 0) {
        $this->$attr = (string) $val;
        continue;
      }

      switch ($attr) {
        case 'urls':
          # code...
          break;
        case 'classloader':
          # code...
          break;
        case 'requires':
          # code...
          break;
        default:
          # code...
          break;
      }
    }
```